cmake/EthCcache.cmake

```diff
@@ -3,13 +3,13 @@
 # The ccache is auto-enabled if the tool is found.
 # To disable set -DCCACHE=OFF option.
 if(NOT DEFINED CMAKE_CXX_COMPILER_LAUNCHER)
-    find_program(CCACHE ccache DOC "ccache tool path; set to OFF to disable")
-    if(CCACHE)
-        set(CMAKE_CXX_COMPILER_LAUNCHER ${CCACHE})
-        if(COMMAND cotire)
-            # Change ccache config to meet cotire requirements.
-            set(ENV{CCACHE_SLOPPINESS} pch_defines,time_macros)
-        endif()
-        message(STATUS "[ccache] Enabled: ${CCACHE}")
+  find_program(CCACHE ccache DOC "ccache tool path; set to OFF to disable")
```

Comment:
There whitespace only changes.

Reply:
Yes. This was accepted this way in solidity. I want this file to be identical to the one from solidity.

Using indenting of 2 spaces (as suggested by Hunter) was probably a mistake...